### PR TITLE
[website] Improve style prop display

### PR DESCRIPTION
**Pull request by vjeux**
- Display the description when it exists
- Display the platform

screen shot 2016-01-24 at 1 37 45 pm

### CODE REVIEW

File: website/layout/AutodocsLayout.js [hunk 1]
```diff
@@ -121,17 +170,10 @@ var ComponentDoc = React.createClass({
...
-              {' '}
-              {style.props[name].type && <span className="propType">
-                {renderType(style.props[name].type)}
-              </span>}
-            </h6>
-          </div>
-        )}
+        {Object.keys(style.props)
+          .sort(sortByPlatform.bind(null, style.props))
+          .map((name) => this.renderStylesheetProp(name, style.props[name]))
```
Comment (fkling) [inline website/layout/AutodocsLayout.js]: `(name)` could be just `name` (not sure what the style conventions are).

Comment (vjeux) [inline website/layout/AutodocsLayout.js]: Not sure either :P I'm going to leave it as it was but if we have a convention at fb, happy to follow it

**Comment by vjeux** [comment]
@facebook-github-bot shipit

**Comment by skevy** [comment]
@vjeux import failed?

**Comment by vjeux** [comment]
Yes, there was a SEV yesterday. I just restarted it

**Pull request closed without merging**